docs/INSTALL.md

```diff
@@ -12,9 +12,11 @@
 
 ## Deploy the Operator
 
-Deploy the operator by running the following command.
+Deploy the operator by running the following command where `<RELEASE_VERSION>` needs to be substituted
+with the desired [release tag](https://github.com/confidential-containers/operator/tags).
+
 ```
-kubectl apply -f https://raw.githubusercontent.com/confidential-containers/operator/main/deploy/deploy.yaml
+kubectl apply -k github.com/confidential-containers/operator/config/release?ref=v0.2.0
```

Comment:
I'm slightly confused by this. The above description seems to imply that `<RELEASE_VERSION>` should be here e.g. `kubectl apply -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>`, but we have a specific example for this release? I think the example is good, but maybe we need to update the description?

Reply:
Any suggestion on the wording, @stevenhorsman? :-)

Reply:
Poor idea to pick the English person to write some English words - as you are all much better than me at it!
My starter for 10 would be:

## Deploy the Operator

Deploy the operator by running the following command where `<RELEASE_VERSION>` needs to be substituted
with the desired [release tag](https://github.com/confidential-containers/operator/tags).
```
kubectl apply -k github.com/confidential-containers/operator/config/release?ref=<RELEASE_VERSION>
```
For example, to deploy the `v0.2.0` release run:
```
kubectl apply -k github.com/confidential-containers/operator/config/release?ref=v0.2.0
```
The operator deploys all resources under `confidential-containers-system` namespace.
...

Reply:
The above suggestion from @stevenhorsman looks fine..